### DISCUSSION TITLE
UVTools - new to the app - What is the meaning of recommendations like 15/2s

### DISCUSSION THREAD

**Discussion by Steve566074** [sn4k3/UVtools#680] [Q&A] [answered]
What is the meaning of recommendations like 15/2s - I understand that 2s means 2 secs but what does the 15 refer to?
![Screenshot 2023-03-24 094347](https://user-images.githubusercontent.com/128797453/227484447-861ca1d0-f9b4-4fd1-a796-c742c924bbb0.png)

**Comment by sn4k3** [comment, marked as answer]
Bottom time / Normal time
15s is for bottom layers.

To know more you can hover the mouse and it will show a tooltip, or access to settings and see fields / description.
Right click also offer an better explanation of the method.
